src/team_lib/mhba/xccl_mhba_mkeys.c

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2020.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#include "xccl_mhba_lib.h"
+
+
+//todo extract to helper header
+int poll_cq(struct ibv_cq *cq, int expected_wc_num, int *actual_wc_num, struct ibv_wc actual_wc[]) {
```

Comment:
is used anywhere else? if not make it static. if extern then call "xccl_mhba_poll_cq"